<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinanceTrackerHomeWindow</class>
 <widget class="QMainWindow" name="FinanceTrackerHomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="accountNameEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Account Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSource">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Source</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sourceLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Sources</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="sourceTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>471</width>
      <height>431</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTextEdit" name="valueAmountEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteRowButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1274</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
